btr-api/.devcontainer/docker-compose.yml

```diff
@@ -3,8 +3,8 @@ version: '3.8'
 services:
   app:
     build:
-      context: ..
-      dockerfile: .devcontainer/Dockerfile
+      context: ../..
```

Comment:
Also, not sure if this got messed up, or it should be like that.

Reply:
same with this file -- should probably be the same paths as before I think? Try running it and see what happens